packages/react-draggable-dialog/src/components/DraggableDialog/useDraggableDialog.ts

```diff
@@ -17,6 +18,9 @@ import {
 } from './DraggableDialog.types';
 import { getParsedDraggableMargin } from './utils/getParsedDraggableMargin';
 import { restrictToBoundaryModifier } from './utils/restrictToBoundaryModifier';
+import { DraggableDialogContextValue } from '../../contexts/DraggableDialogContext';
+
+const noopPositionChange = () => ({});
```

Comment:
it's not in diff, but on line 47 I guess we could use this noopPositionChange 

`onPositionChange: onPositionChange ?? (() => ({}))` -> `onPositionChange: onPositionChange ?? noopPositionChange`